poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
```

Comment:
I think you should upgrade poetry version

Reply:
fixed in [03a30b8](https://github.com/m3dev/gokart/pull/345/commits/03a30b8db2674c3a9f66450f3718dbdfb15cde3d)